.github/actions/setup-test-environment/action.yml

```diff
@@ -28,3 +28,4 @@ runs:
         sudo systemctl start mysql
         mysql -uroot -proot -e "CREATE DATABASE mapknitter_test;"
         bundle exec rake db:schema:load db:migrate --trace
+        sudo apt-get install -y gdal-bin
```

Comment:
Hmmm, apparently, only this package is needed to run unit and integration tests.

Reply:
Yes that's right - i think we do unit and integration tests which run map exporting processes, but we didn't bother to do that for other kinds of tests which are more focused on application logic rather than these big back-end processes.